frame/bridge/ethereum/relay/src/lib.rs

```diff
@@ -266,41 +346,57 @@ decl_module! {
 			<ReceiptVerifyFee<T>>::put(new);
 		}
 
-		/// Remove the specific malicous block
+		/// Remove the specific malicous relay parcel
 		#[weight = 100_000_000]
-		pub fn remove_confirmed_block(origin, number: EthereumBlockNumber) {
+		pub fn remove_relaied_parcel_of(origin, relaied_block_number: EthereumBlockNumber) {
```

Comment:
```suggestion
		pub fn remove_confirmed_parcel_of(origin, relaied_block_number: EthereumBlockNumber) {
```

```Confirmed``` is more straight forward word